check-cla/action.yml

```diff
@@ -102,6 +102,34 @@ runs:
               name: '${{ inputs.label }}'
             })
 
+    # checkout conda/infrastructure to update .clabot
+    - uses: actions/checkout@v3
+      if: steps.contributors.outputs.hasSigned == 'false'
+      with:
+        repository: conda/infrastructure
+
+    # update .clabot
+    - shell: python
+      if: steps.contributors.outputs.hasSigned == 'false'
+      run: |
+        import json
+        from pathlib import Path
+
+        path = Path(".clabot")
+        clabot = json.loads(path.read_text())
+        clabot["contributors"].append("${{ github.actor }}")
+        path.write_text(json.dumps(clabot))
+
+    # create PR
+    - uses: peter-evans/create-pull-request@v4
```

Comment:
Does the action have enough permissions to actually open a PR in another repo?

Reply:
The org token may need to be updated but it may be better to limit the scope and make a new infra-only token for the PR creation.

Reply:
A new token was added for the elevated permissions needed to open a PR in conda/infrastructure.